### ISSUE TITLE
Cannot run `yarn site`

### ISSUE THREAD

**Issue by kanitw** [vega/vega-lite#4431]
```
node_modules/vega-lite/build/src/util.d.ts:1:29 - error TS7016: Could not find a declaration file for module 'json-stable-stringify'. '/Users/kanitw/Documents/code/vega-lite/node_modules/json-stable-stringify/index.js' implicitly has an 'any' type.
  Try `npm install @types/json-stable-stringify` if it exists or add a new declaration (.d.ts) file containing `declare module 'json-stable-stringify';`

1 import stableStringify from 'json-stable-stringify';
                              ~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error.
```

**Comment by domoritz** [comment]
Hmm, the problem is that the site uses an old version of Vega-Lite. This will be fixed with the next release, I think. 

I don't know who's pulling in vega-lite in the site.

**Comment by domoritz** [comment]
So I'd say the fix is to make a new release.

**Comment by kanitw** [comment]
We already released a new version, but I still run into this problem.

**Comment by domoritz** [comment]
I just sent a PR to fix it.

**Comment by kanitw** [comment]
Fixed in $#4469
